interview-questions.md

```diff
@@ -5,7 +5,7 @@ Answer the following questions.
 First, without external resources. Challenge yourself to answer from memory.
 
 Then, research the question to expand on your answer. Even if you feel you have answered the question completely on your own, there is always something more to learn. Write your researched answer in your OWN WORDS.
-
```

Comment:
Interview Questions:
I love that you are elaborating on your interview questions; it is really allowing you to show the interviewer your knowledge! Great work!